util/term/prompt.go

```diff
@@ -53,15 +53,15 @@ func PromptPassword(prompt string, allowEmpty bool) (string, error) {
 
 func Prompt(prompt string) (string, error) {
 	fmt.Print(prompt)
+
 	reader := bufio.NewReader(os.Stdin)
 	text, err := reader.ReadString('\n')
 
 	if err != nil {
 		return "", err
 	}
 
-	text = strings.Replace(text, "\n", "", -1)
+	text = strings.TrimSpace(text)
```

Comment:
This still trims spaces at the end of passwords no?

Reply:
Ah, didn't realize i wasn't on main. 

No, this method doesn't trim passwords, as that's handled by a different method. That other method has however historically called TrimSpace. In the other PR I proposed that if someone wants a space in their password,they should use -p ' my spacey password '

Reply:
Fix this PR so it's off of main, not my other fix branch